composer.json

```diff
@@ -25,11 +25,13 @@
     "jms/serializer": "^1.6|^2.0|^3.0",
 
     "goetas-webservices/xsd2php-runtime": "^0.2.11",
-    "goetas-webservices/soap-common": "^0.2.0"
+    "goetas-webservices/soap-common": "dev-master",
+    "goetas-webservices/soap-reader": "^0.3.3",
+    "goetas-webservices/xsd2php": "dev-master"
```

Comment:
i have released a new version for this https://github.com/goetas-webservices/xsd2php/releases/tag/0.4.7

Reply:
and again, ty :)